tests/api/test_acl_range.py

```diff
@@ -0,0 +1,50 @@
+from pprint import pprint
+
+import pytest
+
+
+class TestSaiAclRange:
+    # object with no parents
+
+    @pytest.mark.dependency(scope='session')
+    def test_acl_range_create(self, npu):
+        commands = [
+            {
+                'name': 'acl_range_1',
+                'op': 'create',
+                'type': 'SAI_OBJECT_TYPE_ACL_RANGE',
+                'attributes': [
+                    'SAI_ACL_RANGE_ATTR_TYPE',
+                    'SAI_ACL_RANGE_TYPE_L4_SRC_PORT_RANGE',
+                    'SAI_ACL_RANGE_ATTR_LIMIT',
+                    '10,20',
+                ],
+            }
+        ]
+
+        results = [*npu.process_commands(commands)]
+        print('======= SAI commands RETURN values create =======')
+        pprint(results)
+        assert all(results), 'Create error'
+
+    def test_acl_range_remove(self, npu):
+        commands = [
+            {
+                'name': 'acl_range_1',
+                'op': 'remove',
+                'type': 'SAI_OBJECT_TYPE_ACL_RANGE',
+                'attributes': [
+                    'SAI_ACL_RANGE_ATTR_TYPE',
+                    'SAI_ACL_RANGE_TYPE_L4_SRC_PORT_RANGE',
+                    'SAI_ACL_RANGE_ATTR_LIMIT',
+                    '10,20',
+                ],
+            }
+        ]
+
+        results = [*npu.process_commands(commands)]
+        print('======= SAI commands RETURN values remove =======')
+        pprint(results)
+        assert all(
```

Comment:
The `process_commands()` API makes `assert` implicitly. The TC will fail in case `result != 'SAI_STATUS_SUCCESS'`. So, it's optional to check the results explicitly.

Reply:
this has been resolved